README.md

```diff
@@ -1 +1,83 @@
-# CytoPipe
\ No newline at end of file
+# Table of contents
+
+- [Table of contents](#table-of-contents)
+- [CytoPipe](#cytopipe)
+- [Formatting](#formatting)
+  - [smkfmt](#smkfmt)
+- [Testing](#testing)
+  - [Dry Runs](#dry-runs)
+
+# CytoPipe
+
+# Formatting
+
+## smkfmt
+
+`snakefmt` is a `snakemake` code formatter that is developed and maintained by the developers of `snakemake`.
+
+In order to use `snakefmt`, `snakemake` must be installed, which can be found here: [snakemake-installation](https://snakemake.readthedocs.io/en/stable/getting_started/installation.html)
+
+`snakefmt` installation processes can be found in the [snakefmt](https://github.com/snakemake/snakefmt#install) repository.
+
+# Testing
+
+## Dry Runs
+
+`Snakemake`'s `dry-runs` is a simple test that ensures that all the required inputs and expected outputs are satisfied. No execution is conducted therefore any potential errors found within the executables **will not be captured.**
```

Comment:
```suggestion
`Snakemake`'s `dry-runs` is a simple test that ensures that all the required inputs and expected outputs are satisfied.
No execution is conducted, therefore the dry run **will not capture** any potential errors found within the executables.
```

Two suggestions baked in here:

1. Please use [one sentence per line](https://news.ycombinator.com/item?id=4642395)
2. Edited second sentence from passive to active voice :)